### ISSUE TITLE
feill to slice the file

### ISSUE THREAD

**Issue by EffectStyle** [Ultimaker/Cura#13094]
### Application Version

5.10

### Platform

Windows 11 

### Printer

prusa mini

### Reproduction steps

i press slice i have not change anything all the other stl no problem whit the same setting 

### Actual results

start to think get to about 30 % blu and then when back to 0 after 5 min 
i close it and then try one more time in 10 min show my the report function

### Expected results

i do not know mb is just file was to big on the prusaslicer show up 12 h 
but i have try on other files that takes full day whit no problems 

### Checklist of files to include

- [X] Log file
- [X] Project file

### Additional information & file uploads

[cura.log](https://github.com/Ultimaker/Cura/files/9396658/cura.log)
[OPM_1553-15-_15_body_top_1_L_FDM (repaired).zip](https://github.com/Ultimaker/Cura/files/9396646/OPM_1553-15-_15_body_top_1_L_FDM.repaired.zip)

**Comment by MariMakes** [comment]
Hey @EffectStyle,

Welcome to GitHub! Thanks for your report. 🎉 

I'm having some trouble reproducing it on my side. 😞 
I'm able to succesfully slice on standard prusa printers.
![image](https://user-images.githubusercontent.com/40423138/186926036-ad506dae-7598-4a30-b2c5-bc9fc10fb78f.png)

Can you please share your configuration file? Than I can see what settings you have for your Prusa Mini. 
If you click Help, Show Configuration Folder, you'll see a bunch of files.
Can you zip those for me and share it here. Don't forget to open Github in your browser to share files. 

Thank you! 👼

**Comment by EffectStyle** [comment]
[cura.zip](https://github.com/Ultimaker/Cura/files/9434146/cura.zip)
[gamebody.zip](https://github.com/Ultimaker/Cura/files/9434187/gamebody.zip)
[MINI_cura_profiles.zip](https://github.com/Ultimaker/Cura/files/9434185/MINI_cura_profiles.zip)
i have use the default from the prusa side 
https://help.prusa3d.com/guide/how-to-import-profiles-to-cura-4-x-windows-macos_17848#17856&_ga=2.259925890.944902336.1661525381-1530490878.1648834170

**Comment by MariMakes** [comment]
Hey @EffectStyle,

Thanks for the extra info and files.
I've been able to recreate the crash, so I'll bring it up to the team to analyze. 
Fingers crossed 🤞 

I also checked your settings and it seems that if you change your layerheight slightly, you'll have a succesful slice!
![image](https://user-images.githubusercontent.com/40423138/187221467-6daf3c3d-44dd-4061-9ccd-25ec8f5ca623.png)
Hopefully you are not stuck anymore.
